cves/kernel/CVE-2013-1763.yml

```diff
@@ -320,8 +335,9 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The commit was signed off by multiple developers.
```

Comment:
maybe include this too https://www.openwall.com/lists/oss-security/2013/02/25/12.  He seems to be vouching for others

Reply:
That link is to a discussion related to which kernel versions are affected by the vulnerability. His reply is not vouching for the others' work, but he is just resolving the discussion.